imports/plugins/core/detailed-analytics/client/templates/actionableAnalytics.html

```diff
@@ -0,0 +1,40 @@
+<template name="actionableAnalytics">
+    <div class="container">
+			<h1 class="text-center"> Actionable Analytics </h1>
+			<hr />
+		<div class="row">
+        <div class="analytics-tab-container">
+            <div class="col-lg-2 col-md-2 col-sm-2 col-xs-2 analytics-tab-menu">
+              <div class="list-group">
+                <a href="#" class="list-group-item active text-center">
+                  <h4 class="fa fa-bar-chart-o"></h4><br/> Total Products Sold
+                </a>
+                <a href="#" class="list-group-item text-center">
+                  <h4 class="fa fa-pie-chart"></h4><br/> Users by UserType
```

Comment:
I thought we said we should look for something more appropriate here and not users by UserType